pandas/core/ops.py

```diff
@@ -1086,7 +1089,7 @@ def f(self, other, axis=default_axis, level=None, fill_value=None):
                 raise ValueError("Incompatible argument shape: %s" %
                                  (other.shape, ))
         else:
-            return self._combine_const(other, na_op)
+            return self._combine_const(other, na_op, fill_value=fill_value)
```

Comment:
How about use `fillna` here, instead of modifying the `_combine_const` method? @jreback